app/routes/docs.notices.unifiedschema.mdx

```diff
@@ -0,0 +1,191 @@
+          allOf: [
+            {
+              $ref: '/schema/gcn/notices/core/Alert.schema.json',
+            },
+            {
+              $ref: '/schema/gcn/notices/core/Event.schema.json',
+            },
+            {
+              $ref: '/schema/gcn/notices/core/DateTime.schema.json',
+            },
+            {
+              $ref: '/schema/gcn/notices/core/Localization.schema.json',
+            },
+            {
+              $ref: '/schema/gcn/notices/core/Statistics.schema.json',
+            },
+            {
+              $ref: '/schema/gcn/notices/core/Reporter.schema.json',
+            },
+            {
+              $ref: '/schema/gcn/notices/core/AdditionalInfo.schema.json',
+            },
+          ],
+          properties: {
+            $schema: true,
+            follow_up_event: {
+              type: 'string',
+              description:
+                'Name or trigger time of the external trigger that launched the search.',
+            },
+            follow_up_type: {
+              type: 'string',
+              description:
+                'Type of external trigger that launched the search, eg GW, neutrino, etc.',
+            },
+          },
+          unevaluatedProperties: false,
+        },
+        null,
+        2
+      )}
+    />
+  </Tab>
+  <Tab label="Alert.example.json">
+    <Highlight
+      language="json"
+      code={JSON.stringify(
+        {
+          $schema:
+            'https://gcn.nasa.gov/schema/main/gcn/notices/swift/bat/guano/alert.schema.json',
+          mission: 'Swift',
+          instrument: 'BAT-GUANO',
+          messenger: 'EM',
+          record_number: 1,
+          alert_datetime: '2023-01-01T03:24:36.0Z',
+          alert_tense: 'current',
+          alert_type: 'initial',
+          trigger_time: '2022-12-31T21:46:05.13Z',
+          rate_snr: 15.8,
+          rate_duration: 0.256,
+          rate_energy_range: [15, 350],
+          classification: { GRB: 1 },
+          far: 0.00001,
+          follow_up_event: 'Fermi 694215970',
+          follow_up_type: 'GRB',
+          data_archive_page:
+            'https://guano.swift.psu.edu/trigger_report?id=694215995',
+          id: ['694215995'],
+        },
+        null,
+        2
+      )}
+    />
+  </Tab>
+</Tabs>
+
+While defining JSON schema structures, please don't hesitate to [reach out](https://gcn.nasa.gov/contact) with any
+questions about Schema field definitions or specific fields you are looking for.
+
+## Validating Schema
+
+A validation pre-commit hook has been configured for this project. When you commit your updates, the validator will run to make sure that all of the schemas and their respective examples function correctly. This step also checks general formatting using the [Prettier](https://prettier.io/) formatter. To quickly format your code, run the following command in the root directory of your local repository. This will format spacing, line-breaks, etc. as needed.
```

Comment:
```suggestion
The Git repository is configured to automatically format and validate your code at the time that you commit it. This means when you commit your updates, the validator will run to make sure that all of the schemas and their respective examples function correctly. This step also checks general formatting using the [Prettier](https://prettier.io/) formatter. To quickly format your code, run the following command in the root directory of your local repository. This will format spacing, line-breaks, etc. as needed.
```